test/test_sparql/test_sparql.py

```diff
@@ -90,6 +90,36 @@ def test_sparql_bnodelist():
     prepareQuery("select * where { ?s ?p ( [] [ ?p2 ?o2 ] [] ) . }")
 
 
+@pytest.mark.xfail
+def test_sparql_polist():
+    """
+
+    syntax tests for equivalence object and predicate-object lists
+
+    """
+
+    g = Graph()
+    g.parse(
+        data="""
+    @prefix : <urn:ns1:> .
+    :s :p [ :v 1 ], [ :v 2].
+    """,
+        format="n3",
```

Comment:
```suggestion
        format="n3",
```

If possible we should avoid using n3 in tests unless absolutely needed, in part because it is not standardized, and further it is also not exactly RDF, N3 allows encoding things that cannot be represented in any other RDF formats, and that is not in line with RDF abstract syntax.

Reply:
Fixed this with some other minor things.